core/src/main/java/org/bitcoinj/core/SendDsq.java

```diff
@@ -1,12 +1,40 @@
 package org.bitcoinj.core;
 
+import java.io.IOException;
+import java.io.OutputStream;
+
 /**
  * Created by Hash Engineering on 4/12/2019.
  */
-public class SendDsq extends EmptyMessage {
+public class SendDsq extends Message {
```

Comment:
Should we move this to `/coinjoin` as well?

Reply:
Let's move it to the `conjoin` package.

Reply:
Perhaps we should also rename it to `SendCoinJoinQueue` so it will align with `CoinJoinQueue`